composer.json

```
@@ -14,7 +14,7 @@
     },
     "require-dev": {
         "doctrine/cache": "^1.11",
-        "friendsofphp/php-cs-fixer": "^2.14",
+        "friendsofphp/php-cs-fixer": "^3.47.0",
```

```suggestion
        "friendsofphp/php-cs-fixer": "^2.19 || ^3.48.0",
```

We need to allow v2 or else Composer won't be able to calculate a valid set of dependencies for PHP 7.2. That being said, we probably don't need to run CS Fixer on _every_ PHP version. Maybe move the CS Fixer check into a separate job running on PHP 8.3?